LiskHQ/lisk-mobile
# Migrate SignIn/BioAuth component to new design - Closes #751

# What was the bug or feature?
Described in #751

### How did I fix it?
- Moved (and cleaned up) SignIn/BiometricAuth component to a separate folder to reduce the complexity in form/styles.js file
- Extracted createdAccount part of SignIn/Form component to be a shared
- Updated SignIn/BiometricAuth component to reflect new design

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
- Sign In with BioAuth then enable keeping passphrase in keychain.
- Close & Reopen the app.
- Observe the changes in this screen.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
